README.md

```diff
@@ -1,5 +1,13 @@
 # GitHub Action for building Azure Web Apps
 
+![V1_BuildAndDeploy_PythonApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_PythonApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_PhpApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_PhpApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_NodeApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_NodeApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp/badge.svg?branch=master)
+![V2Beta_BuildAndDeploy_PythonApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V2Beta_BuildAndDeploy_PythonApp_ToAzureWebApp/badge.svg?branch=master)

```

Comment:
All the v2-beta ones should be based on `v2-beta` branch?

Reply:
These is master branch under githubaction-samples repo I think.

Reply:
You are right. Thanks.

Reply:
I'm also thinking if there's a way to separate v1 and v2-beta maybe? right now, it'll send dispatch event to all sample apps not matter which branch it updated. If we could send dispatch event to sample apps according to v1 or v2-beta branch separately, that makes sense better.